ra_aid/__main__.py

```
@@ -1477,6 +1555,47 @@ def main():
+                    from ra_aid.agents.supervisor import run_supervisor
+
+                    result = run_supervisor(
+                        user_request=args.message or "",
+                        expert_model=expert_model,
+                        worker_model=worker_model,
+                    )
+
+                    # Print final result
+                    print()
+                    console.print(Panel(
+                        Markdown(result),
```

Ruff (0.14.8)

Undefined name `Markdown`

(F821)